### ISSUE TITLE
ember-data dependency

### ISSUE THREAD

**Issue by wmlele** [funkensturm/ember-local-storage#35]
I am using ember-local-storage in a project that's *not* using ember-data. 
Since 0.0.7 is now failing at runtime ("DS is not defined"). 
In 0.0.9, build is also failing with an obscure error ('Invalid Version: undefined') when ember-data is not present.

Is the strict dependency on ember-data a long term requirement, or is just a temporary accident?

**Comment by fsmanuel** [comment]
@wmlele that's weird. Since 0.0.9 it shouldn't depend on ember-data. Do you have a sample app?

**Comment by wmlele** [comment]
ember-cli 1.13.8, node 0.12.7, linux-x86_64:
```
wml@miranda:/storage/wml/workspaces/cm> ember new testapp
version: 1.13.8
installing app
  create .bowerrc
  [...]

wml@miranda:/storage/wml/workspaces/cm> cd testapp

wml@miranda:/storage/wml/workspaces/cm/testapp> npm uninstall --save-dev ember-data
unbuild ember-data@1.13.8

wml@miranda:/storage/wml/workspaces/cm/testapp> bower uninstall --save ember-data
bower uninstall     ember-data

wml@miranda:/storage/wml/workspaces/cm/testapp> npm install --save-dev ember-local-storage
ember-local-storage@0.0.9 node_modules/ember-local-storage
├── ember-cli-version-checker@1.1.4 (semver@4.3.6)
└── broccoli-stew@0.3.5 (walk-sync@0.1.3, debug@2.2.0, chalk@0.5.1, rsvp@3.1.0, minimatch@2.0.10, cauliflower-filter@1.1.0, fs-extra@0.13.0, broccoli-merge-trees@0.2.3, broccoli-funnel@0.2.8, lodash-node@2.4.1)

wml@miranda:/storage/wml/workspaces/cm/testapp> ember s
version: 1.13.8
Invalid Version: undefined
TypeError: Invalid Version: undefined
    at new SemVer (/storage/wml/workspaces/cm/testapp/node_modules/ember-local-storage/node_modules/ember-cli-version-checker/node_modules/semver/semver.js:281:11)
    at compare (/storage/wml/workspaces/cm/testapp/node_modules/ember-local-storage/node_modules/ember-cli-version-checker/node_modules/semver/semver.js:571:10)
    at Function.gt (/storage/wml/workspaces/cm/testapp/node_modules/ember-local-storage/node_modules/ember-cli-version-checker/node_modules/semver/semver.js:600:10)
    at DependencyVersionChecker.(anonymous function) [as gt] (/storage/wml/workspaces/cm/testapp/node_modules/ember-local-storage/node_modules/ember-cli-version-checker/index.js:68:26)
    at Class.included (/storage/wml/workspaces/cm/testapp/node_modules/ember-local-storage/index.js:15:29)
    at EmberApp.<anonymous> (/storage/wml/workspaces/cm/testapp/node_modules/ember-cli/lib/broccoli/ember-app.js:328:15)
    at Array.filter (native)
    at EmberApp._notifyAddonIncluded (/storage/wml/workspaces/cm/testapp/node_modules/ember-cli/lib/broccoli/ember-app.js:323:45)
    at new EmberApp (/storage/wml/workspaces/cm/testapp/node_modules/ember-cli/lib/broccoli/ember-app.js:118:8)
    at module.exports (/storage/wml/workspaces/cm/testapp/ember-cli-build.js:5:13)
```

**Comment by fsmanuel** [comment]
I'll try to reproduce and fix it.

**Comment by wmlele** [comment]
```
npm install --save-dev ember-local-storage@0.0.8 
```

would build, but gives, at runtime:

```
Uncaught ReferenceError: DS is not defined(anonymous function) @ app-shims.js:15mod.state @ loader.js:141tryFinally @ loader.js:21requireModule @ loader.js:139requireFrom @ loader.js:112reify @ loader.js:97mod.state @ loader.js:140tryFinally @ loader.js:21requireModule @ loader.js:139requireFrom @ loader.js:112reify @ loader.js:97mod.state @ loader.js:140tryFinally @ loader.js:21requireModule @ loader.js:139(anonymous function) @ ember-load-initializers.js:24default @ ember-load-initializers.js:21(anonymous function) @ app.js:15mod.state @ loader.js:141tryFinally @ loader.js:21requireModule @ loader.js:139(anonymous function) @ app-boot.js:25
```

**Comment by fsmanuel** [comment]
@wmlele i published 0.0.10 that should fix the issue.
